net-mail/dot-forward/dot-forward-0.71-r4.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-inherit fixheadtails qmail
+inherit fixheadtails flag-o-matic qmail
```

Comment:
Missing revbump (for the implicit func. decl stuff, we don't want to take any risks).

But also, while we don't normally revbump for "build time fixes", the nature of a build-time fix is kind of varied. a huge patch like this merits a revbump anyway (even if there were no expected possible change in runtime behaviour) just because there's a non-zero chance it'll affect behaviour or cause a bug we want to know about sooner rather than later.